sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -371,28 +371,44 @@ case class GpuFilterExec(condition: Expression, child: SparkPlan)
 /**
  * Physical plan for range (generating a range of 64 bit numbers).
  */
-case class GpuRangeExec(range: org.apache.spark.sql.catalyst.plans.logical.Range,
+case class GpuRangeExec(
```

Comment:
this one seems different than other changes in this PR. Just curious on why the change.

Reply:
I inlined `logical.Range` into it. It is not needed, but made the code a little more robust in my opinion. I can remove it if we want to.

Reply:
no that's fine just curious